### ISSUE TITLE
Accessibility: Screen reader friendly

### ISSUE THREAD

**Issue by SunderB** [SunderB/sonic-pi-phoenix-website#7]
The website should be accessible to those who use a screen reader. This includes both reading and interacting with the website. 

## Focus order
The focus order of elements on the age should be logical, so that screen readers read the content in the intended order.

WCAG 2.1 Guideline 2.4.3 (Level A):
> **If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.**

## Alt text
All images should have suitable alt text that describes what the picture is showing, or any important information conveyed by it.

Note: This is more applicable for the tutorial pages, and changes to the alt text would probably be handled in the original tutorial files in the main sonic-pi repo.

WCAG 2.1 Guideline 1.1.1 (Level A):
> **All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for the situations listed below.**
>
>Controls, Input: If non-text content is a control or accepts user input, then it has a name that describes its purpose. (Refer to Success Criterion 4.1.2 for additional requirements for controls and content that accepts user input.)
>
>Time-Based Media: If non-text content is time-based media, then text alternatives at least provide descriptive identification of the non-text content. (Refer to Guideline 1.2 for additional requirements for media.)
>
> Test: If non-text content is a test or exercise that would be invalid if presented in text, then text alternatives at least provide descriptive identification of the non-text content.
>
> Sensory: If non-text content is primarily intended to create a specific sensory experience, then text alternatives at least provide descriptive identification of the non-text content.
>
> CAPTCHA: If the purpose of non-text content is to confirm that content is being accessed by a person rather than a computer, then text alternatives that identify and describe the purpose of the non-text content are provided, and alternative forms of CAPTCHA using output modes for different types of sensory perception are provided to accommodate different disabilities.
>
>Decoration, Formatting, Invisible: If non-text content is pure decoration, is used only for visual formatting, or is not presented to users, then it is implemented in a way that it can be ignored by assistive technology.
